doc/user_guide/transformations.rst

```diff
@@ -707,8 +707,11 @@ The current available options are:
 |              | in a work-group execution (kernel instances |         |
 |              | executed at the same time).                 |         |
 +--------------+---------------------------------------------+---------+
-| queue_number | The identifier of the OpenCL Command Queue  | 1       |
-|              | to which the kernel should be submitted.    |         |
+| queue_number | The identifier of the OpenCL command_queue  | 1       |
+|              | to which the kernel should be submitted. If |         |
+|              | the kernel has a dependency to a different  |         |
```

Comment:
"...dependency on another kernel submitted to a different..."

Reply:
Done